testing/fixtures/fixtures.go

```diff
@@ -73,3 +73,36 @@ func Clone(baseDir string, projects []Project) error {
 
 	return nil
 }
+
+// ProjectInitializerFunction defines how a single project should be initialized *after* it has already been cloned
+type ProjectInitializerFunction func(proj Project, projectDir string) error
+
+// InitializeProjects initializes provided projects in parallel using the provided initialization function. Errors are returned as a map of the Project.Name to the error thrown
+func InitializeProjects(testDir string, projects []Project, projectInitializer ProjectInitializerFunction) map[string]error {
```

Comment:
Why not fold this into `Clone` and rename that to `Initialize`? For fixtures that should _only_ be cloned, we can pass a no-op.

Which of these choices is more ergonomic depends on the kinds of fixtures you expect to work with. Will the majority of fixtures need initialization? (I think so.)